### ISSUE TITLE
Clarification on DMI_Dynamics_T

### ISSUE THREAD

**Issue by christianstahl** [openETCS/modeling#120]
There exists an element sup_status in the above type definition, which either is wrong or has a misleading name. The values of the enumerator sup_status (RSM, TSM etc) refer to types of speed and distance monitoring, see SRS 3.13.10. A status of a type of speed and distance monitoring is, for example, normal status, warning status or intervention status.
Q1: Do you expect a type or a status?
Q2: What is PIM?
Q3: If a type is expected (e.g. RSM), does the DMI also need the information about the current status (e.g. warningstatus)?
Q4: The element m_warning seems to refer to the warning status, right?
@BerndHekele 
@vgontier

**Comment by BerndHekele** [comment]
Dear Christian,  
here come some information for claification:  

Q1: in this package a status is sent to the DMI. However, I would like to split responsibilities in a way that dynamics is sent  by the DMI on a regular schedule and collects "event-like" information for that purpose. I will clarify this statement and send an update.

Q2, Q3: comes from the ERSA implementation. We will claify the usecase.

Q4: correct

**Comment by BerndHekele** [comment]
Dear Christian,
I proceeded with the anaöysis of this function. Now, we can continue the discussion.

Q1: this information is used in an periodic update. So, information is of quality "status" and should e provided continuosly.
Q2: taken from the ERSA approach - no further information available. I believe, it is not relevant for our use-case.
Q3: I defined an interface for your input (see issue#134).  
M_Warning is a situation where an violation of the speed range has to be indicated.

Is this information sufficient to cover your use-cases with the DMI?
Your interface is defined here: manage_DMI_Output_Pkg::speedSupervisionForDMI_T
